pvlib/test/test_solarposition.py

```diff
@@ -15,6 +16,9 @@
 from conftest import (requires_ephem, needs_pandas_0_17,
                       requires_spa_c, requires_numba)
 
+DIRNAME = os.path.dirname(__file__)
+PROJDIR = os.path.dirname(DIRNAME)
+DATADIR = os.path.join(PROJDIR, 'data')
```

Comment:
probably better to put these in the function if they're not going to be reused elsewhere.